src/distrs/gamma.jl

```diff
@@ -1,6 +1,6 @@
 # functions related to gamma distribution
 
-using HypergeometricFunctions: drummond1F1
+using HypergeometricFunctions: _₁F₁
```

Comment:
I assume we might have to update the compat entry for HypergeometricFunctions?

Reply:
It looks like it was introduced in 0.3.7, https://github.com/JuliaMath/HypergeometricFunctions.jl/commit/251d4221b7d6dccbe00f554948ac582299cee10d, so the current bound should be sufficient.